packages/core/src/Stores/client-store.js

```diff
@@ -741,6 +763,11 @@ export default class ClientStore extends BaseStore {
         return this.isBotAllowed();
     }
 
+    @computed
+    get active_account_landing_company() {
+        return this.is_logged_in ? this.getAccount().landing_company_shortcode : null;
+    }
+
```

Comment:
Do we need this? we already have a function `get landing_company_shortcode()` to get the current landing company of current loginid. Cant we reuse that ?